conformance_tests/sysman/test_sysman_init/src/test_init_sysman_enum_freq_with_core_handle.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#include "gtest/gtest.h"
+#include "logging/logging.hpp"
+#include "utils/utils.hpp"
+#include "test_harness/test_harness.hpp"
+#include <level_zero/zes_api.h>
+
+namespace {
+
+TEST(
+    SysmanInitTests,
+    GivenSysmanInitializedFromZesInitAndCoreInitializedWithSysmanFlagWhenzesEnumFrequencyDomainsIsCalledWithCoreHandleThenUninitializedErrorIsReturned) {
+  static char sys_env[] = "ZES_ENABLE_SYSMAN=1";
```

Comment:
Could you add another test similar to this but where `ZES_ENABLE_SYSMAN=0`
Expectation would be that we ensure sysman APIs cannot be accessed using core handles and no crash is observed in this use case as well.